### ISSUE TITLE
Reading from file to NamedArray

### ISSUE THREAD

**Issue by Mastomaki** [davidavdav/NamedArrays.jl#109]
Documentation should be added about the best ways to read from text file to NamedArray. My current plan is to first read a DataFrame via **CSV.jl** and then convert it using the function provided by [dietercastel](https://github.com/davidavdav/NamedArrays.jl/pull/96).

**Comment by davidavdav** [comment]
Yes, documentation.  I have to study how that works.  Do you know of a recommended and hosted platform for that?

**Comment by Mastomaki** [comment]
Not really. I believe the documentation of registered packages appears in https://juliapackages.com/ if it is present in the Github repository. And [documenter.jl](https://github.com/JuliaDocs/Documenter.jl) can be used to make documentation.

**Comment by sciencepeak** [comment]
> 
> 
> Yes, documentation. I have to study how that works. Do you know of a recommended and hosted platform for that?

I think it is not necessary to master documenter.jl to write a formal, perfect documentation. If the usage of conversion between NamedArray and DataFrame can be added to the ReadMe file of this repository, that is good enough for now for people to learn it.

I think your package is very important for Julia to attract data science users from Python Pandas and R, where data frame and matrix can easily be converted to each other without losing row names and column names. Thanks a lot for your work.
